tests/phpunit/tests/option/networkOption.php

```diff
@@ -208,24 +278,140 @@ public function test_update_network_option_array_with_object() {
 			),
 		);
 
-		$array_w_object_2 = array(
-			'url'       => 'http://src.wordpress-develop.dev/wp-content/uploads/2016/10/cropped-Blurry-Lights.jpg',
-			'meta_data' => (object) array(
-				'attachment_id' => 292,
-				'height'        => 708,
-				'width'         => 1260,
-			),
-		);
+		add_metadata( 'site', $network_id, $option, $array_w_object, true );
+		$this->assertSame( $array_w_object, get_network_option( $network_id, $option ) );
```

Comment:
We should use `assertEquals` instead of `assertSame` similar to `test_get_scheduled_event_singles()` here https://github.com/WordPress/wordpress-develop/blob/trunk/tests/phpunit/tests/cron.php#L602-L635 because object key value does not match.

```suggestion
		$this->assertEquals( $array_w_object, get_network_option( $network_id, $option ) );
```